framework/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Yii Framework 2 Change Log
 - Enh #10158: Added the possibility to specify CSS and Javascript options per file in `\yii\web\AssetBundle` (machour)
 - Enh #10170: `Yii::powered()` now uses `Yii::t()` (SamMousa)
 - Enh: Added last resort measure for `FileHelper::removeDirectory()` fail to unlink symlinks under Windows (samdark)
+- Enh #9656: All reading / writing to AR attributes via a single path. (SamMousa)
```

Comment:
It's unclear as for me.
What about `BaseActiveRecord refactored to get, set and unset attributes values using the same approach`?